util/ci/build.sh

```diff
@@ -1,8 +1,8 @@
 #! /bin/bash -e
 
-mkdir cmakebuild
-cd cmakebuild
+mkdir build
+cd build
 cmake -DCMAKE_BUILD_TYPE=Debug \
 	-DRUN_IN_PLACE=TRUE -DENABLE_GETTEXT=TRUE \
 	-DBUILD_SERVER=TRUE ${CMAKE_FLAGS} ..
-make -j2
+make -j$(nproc)
```

Comment:
Regarding this I believe the CI has 1 core but 2 was chosen for speed just like you suspected.

Reply:
well this is inconclusive:
![Unbenannt](https://user-images.githubusercontent.com/1042418/151701159-31a4c324-0911-4881-be02-8256d5f21778.png)